src/transaction/log_replication.cpp

```diff
@@ -438,12 +438,23 @@ namespace cublog
     return m_most_recent_trantable_snapshot_lsa.load ();
   }
 
-  log_lsa replicator::get_redo_lsa () const
+  log_lsa replicator::get_highest_processed_lsa () const
```

Comment:
in `replicator::wait_past_target_lsa` add a TODO comment that it needs to be refactored to take into account the two flavors once atomic replication is added